pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -309,6 +321,19 @@ func newKubeClientset(configPath string) (*kubernetes.Clientset, error) {
 	return client, nil
 }
 
+func newAggregatorClientset(configPath string) (*aggregator.Clientset, error) {
+	config, err := getKubeConfig(configPath)
+	if err != nil {
+		return nil, k8sErr.Wrap(err)
```

Comment:
k8sErr.Wrap is called here and in clients of this function, may we just call in caller? (https://github.com/spiffe/spire/pull/2193/files#diff-05bd138f46b350382d622e28168b1dce2b0e896a0a7dfa4b0eba16e1959bd48fR292)